docs/contracts.rst

```diff
@@ -746,39 +746,40 @@ converted to ``uint8``.
 Events
 ******
 
-Events allow the convenient usage of the EVM logging facilities,
-which in turn can be used to "call" JavaScript callbacks in the user interface
-of a dapp, which listen for these events.
+Events allow you to use the EVM logging facilities to "call" JavaScript
+callbacks in the user interface of a dapp, which listen for these events.
 
-Events are
-inheritable members of contracts. When they are called, they cause the
+Events are inheritable members of contracts. When you call them, they cause the
```

Comment:
> ...they cause the arguments to be stored in the transaction's log...

I think there should be clear statement right at this place, that indexed arguments are stored in another structure (topics) than non-indexed arguments.  The wording should clearly state that storage of the arguments depends on their `indexed` flag. I believe typical assumption about indexes is that they actually duplicate stored data in index structures, not that data are removed from the actual storage and moved into index structure which is the only place they exist.

I would rephrase to:

> ...they cause the log entry to be stored in the transaction's log...

And later how are arguments stored within log entry.